### ISSUE TITLE
RADIUS device configuration

### ISSUE THREAD

**Issue by Firstyear** [Firstyear/kanidm#124]
Using ios/macos profiles or android equivalent allow scanning or link generation to be sent to a device to configure it's radius/wifi. This likely relies on a domain info object or type existing to store the realms ssid

**Comment by Firstyear** [comment]
Apparently this may work in QR codes?

```
WIFI:T:WPA2-EAP;S:SSID;E:PEAP;PH2:MSCHAPV2;I:IDENTITY;P:PASSWORD;;
```

**Comment by cuberoot74088** [comment]
I had a look on that and found a stack overflow saying it could create and read such a qrcode. I could not reproduce. Was only able to create qrcodes for wpa2 psk networks. 
Android 11 also allows sharing WiFi settings by generating qrcodes. This works for wpa2 psk. No such option for wpa2 enterprise. 

As far as I understood, the above code is a proposal and probably not adopted/implemented.

Even if a custom qrcodes scanner app could understand, this would required this special app for the user. So, might also install the WiFi by some other means.

I also found a lot of hints around centrally managed devices in corporate settings. In that case, it would be solved as well.

I have not digged deeper for now. Any updates are welcome.

**Comment by Firstyear** [comment]
Yes, we've tried with android and wpa2 enterprise without success. The way this will work for ios is using deployment profiles, so there needs to be a similar thing for android (else we jst can't really do this outside the ios ecosystem).

**Comment by gbw** [comment]
Research into Wi-Fi Easy Connect to see if it's a viable option for onboarding wifi clients using RADIUS. The spec was implemented for Android 10 released in 2019.

**Comment by kri164** [comment]
Looking at LineageOS 22.2 (AOSP Android 15) settings, you can share(export) only WPAx-Personal wifi networks.
There is no SHARE button on WPAx-Enterprise wifi networks. Just FYI.
